src/components/button/Button.component.tsx

```diff
@@ -1,13 +1,30 @@
+import React from 'react';
+
 import { ButtonProps } from '@mui/material/Button';
 
 import { StyledButton } from './Button.styles';
 
 interface CustomButtonProps extends ButtonProps {
     label: string;
+    fontSize?: string;
+    padding?: string;
 }
 
-export const CustomButton = ({ label, ...props }: CustomButtonProps) => (
-    <StyledButton variant="contained" color="primary" {...props}>
+export const CustomButton: React.FC<CustomButtonProps> = ({
```

Comment:
by convention, the main component that gets exported from a file has the same name as the folder and the file. So this component must be named `Button`.

Additionally, we dont use the word `my` or `custom` in front of our components. Change in all places where applicable